### PR TITLE
strict sphinx attestation

### PR DESCRIPTION
**Pull request by kristapratico**
Resolves https://github.com/Azure/azure-sdk-for-python/issues/33521

### CODE REVIEW

File: sdk/attestation/azure-security-attestation/pyproject.toml [hunk 1]
```diff
@@ -4,3 +4,4 @@ pyright = false
 type_check_samples = false
 verifytypes = false
 ci_enabled = false
+strict_sphinx = true
```
Comment (LarryOsterman) [inline sdk/attestation/azure-security-attestation/pyproject.toml:7]: Add a trailing CR?

**Review by LarryOsterman — APPROVED** [review]

**Comment by LarryOsterman** [comment]
Btw, you may need to re-record tests and update the list of live-only tests. The current recording infrastructure sanitizes all JWTs and most of the responses from the attestation service include JWTs.

**Comment by kristapratico** [comment]
> Btw, you may need to re-record tests and update the list of live-only tests. The current recording infrastructure sanitizes all JWTs and most of the responses from the attestation service include JWTs.

I'll leave this up to the library owners, I don't have the context for fixing the other failing checks.

**Pull request merged**